rpcserver.go

```diff
@@ -1099,11 +1099,13 @@ func (r *rpcServer) VerifyProof(ctx context.Context,
 	}
 
 	headerVerifier := tapgarden.GenHeaderVerifier(ctx, r.cfg.ChainBridge)
-	_, err = proofFile.Verify(
-		ctx, headerVerifier,
-	)
+	_, err = proofFile.Verify(ctx, headerVerifier)
 	valid := err == nil
 
+	if !valid {
+		rpcsLog.Errorf("Proof verification failed with err: %v", err)
+	}
+
```

Comment:
This pattern is strange. I think we should get rid of the variable `valid` and this `if` clause can just inspect `err`.

When populating the `Valid` field in the `return` statement, we can just set it to `err == nil`.